src/components/AnnouncementsPage/Announcements.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import Typography from '@material-ui/core/Typography';
+import useTheme from '@material-ui/core/styles/useTheme';
+import Container from '@material-ui/core/Container';
+
+export default function AnnouncementsPage() {
+	const theme = useTheme();
+
+	return (
+		<Container>
```

Comment:
```suggestion
		<Container maxWidth='md'>
```

All the other pages use `md` width containers, so let's make this `md` as well for consistency. This prop and its values can be found [here](https://mui.com/api/container/) on the MUI docs for `Container`

Reply:
ah i see, thanks!